Dockerfile

```diff
@@ -1,6 +1,5 @@
 FROM fedora:latest
-# e2fsprogs -- docker @ F20 wants it
-RUN yum -y install docker git python-docker-py python-setuptools e2fsprogs koji python-pip python-backports-lzma osbs gssproxy
+RUN dnf -y install docker git python-docker-py python-setuptools e2fsprogs koji python-pip python-backports-lzma osbs gssproxy; dnf clean all
```

Comment:
Good catch! Its important: if the packages will fail to install it would run ```clean```, it would succeed and move on